src/MainWindow.cpp

```diff
@@ -12,6 +12,8 @@
 #include "SettingsWindow.h"
 #include "SlayerApp.h"
 #include "AboutWindow.h"
+#include <AboutWindow.h>
+#include <Catalog.h>
```

Comment:
Need to define B_TRANSLATE_CONTEXT

OTOH, why only do the localization for the AboutWindow?

Reply:
@humdingerb do you mean I have to use localization for all the sources???
Or do you mean I shouldn't use localization?

Reply:
Adding localization only makes sense when done for the whole app.

Reply:
@humdingerb Okay.